src/main/java/taskList/TaskList.java

```diff
@@ -0,0 +1,122 @@
+package taskList;
+
+import excceptions.WeiException;
+import tasks.Task;
+
+import java.util.ArrayList;
+
+/**
+ * Represents a collection of tasks.
+ */
+public class TaskList {
+    private ArrayList<Task> tasks;
```

Comment:
The use of an ArrayList<Task> for task storage is appropriate

Reply:
Good idea changing to ArrayList for task storage. Good that you remembered to name it in its plural form.